codespell_lib/data/dictionary.txt

```diff
@@ -55998,6 +56002,7 @@ vlues->values, blues,
 vluetooth->Bluetooth
 vluing->valuing
 voif->void
+voight->voigt
```

Comment:
What does `voigt` mean?

Reply:
Voigt notation is a way to remove redundancy by listing only the actual degrees of freedom in a symmetric matrix.
https://en.wikipedia.org/wiki/Voigt_notation

Reply:
Ah, it's a proper name. We usually enter them with a capital letter in the dictionary (`Voigt`).

Reply:
Done.